po/ro.po

```diff
@@ -0,0 +1,703 @@
+# Paru Romanian translation
+# Copyright (C) 2021
+# This file is distributed under the same license as the paru package.
+# Adrian Costin <adriancostin6@gmail.com>, 2021.
+msgid ""
+msgstr ""
+"Project-Id-Version: paru VERSION\n"
+"Report-Msgid-Bugs-To: https://github.com/Morganamilo/paru\n"
+"POT-Creation-Date: 2021-06-30 21:32+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Adrian Costin <adriancostin6@gmail.com>\n"
+"Language-Team: \n"
+"Language: ro\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: src/clean.rs:31
+msgid "Do you want to remove ALL AUR packages from cache?"
+msgstr "Doriți să stergeți din cache TOATE PACHETELE care provin din AUR?"
+
+#: src/clean.rs:33
+msgid "Do you want to remove all other AUR packages from cache?"
+msgstr "Doriți să stergeți din cache toate celelalte pachete AUR?"
+
+#: src/clean.rs:40
+msgid "Clone Directory: {}"
```

Comment:
Is there no translation for directory?

Reply:
I didn't know how to interpret this, but managed to find a decent translation at last. It is ok now, as far as I can tell. This refers to the directory where the cloned files are stored right?

Reply:
Also fixed the translation for the Diff Directory